Formula/i/ignite.rb

```diff
@@ -0,0 +1,22 @@
+class Ignite < Formula
+  desc "Build, launch, and maintain any crypto application with Ignite CLI"
+  homepage "https://github.com/ignite/cli"
+  url "https://github.com/ignite/cli/archive/refs/tags/v28.2.0.tar.gz"
+  sha256 "556f953fd7f922354dea64e7b3dade5dd75b3f62ece93167e2ba126cac27602e"
+  license "Apache-2.0"
+
+  depends_on "go"
+  depends_on "node"
+
+  def install
+    bin.mkpath
+    system "go", "build", "-mod=readonly", *std_go_args, "-o", bin/"ignite", "./ignite/cmd/ignite/..."
```

Comment:
```suggestion
    system "go", "build", "-mod=readonly", *std_go_args(output:  bin/"ignite), ".../ignite/cmd/ignite"
```

Reply:
@julienrbrt This suggestion needs to be implemented.

Reply:
Please do. I'm happy to help debug if it fails.

Reply:
Same error, I am getting locally on CI due to the switch from `./ignite/cmd/ignite/...` to `.../ignite/cmd/ignite`.
cc @p-linnane